### ISSUE TITLE
Errors with lang

### ISSUE THREAD

**Issue by shmidtelson** [MasterGroosha/telegram-report-bot#5]
Hi! I have settings docker-compose a bit different
I dont use `env_file` but use `environment` section and i think bot cant see my settings, because it sends me 
 ```python
 Incorrect value. Must be one of: en, ru (type=value_error)
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/app/bot/__main__.py", line 8, in <module>
    from bot.before_start import fetch_admins, check_rights_and_permissions
  File "/app/bot/before_start.py", line 6, in <module>
    from bot.config_reader import config
  File "/app/bot/config_reader.py", line 40, in <module>
    config = Settings()
  File "pydantic/env_settings.py", line 38, in pydantic.env_settings.BaseSettings.__init__
  File "pydantic/main.py", line 331, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for Settings
lang
```

**Comment by MasterGroosha** [comment]
Could you please show me your environment section? Or, what's better, a quick guide to help me reproduce your problem.

**Comment by shmidtelson** [comment]
I think that you should use this magic from `python-dotenv` lib
<img width="721" alt="image" src="https://user-images.githubusercontent.com/6944443/167796316-f1e6e475-ff1d-4092-9215-64838f66a3a2.png">

My configuration here 
<img width="749" alt="Screenshot 2022-05-11 at 10 46 53" src="https://user-images.githubusercontent.com/6944443/167796617-ed15bb6c-e0f1-440f-b33c-a511f560c06e.png">

**Comment by MasterGroosha** [comment]
Oh, my bad, could you please replace "BOT_LANGUAGE" variable with "LANG" and check if it works? 
It looks like I've [changed its name](https://github.com/MasterGroosha/telegram-report-bot/blob/aiogram3/bot/config_reader.py#L8) during my switch to pydantic.

I'm not happy with how my settings are structured, so if you have any advice, please let me know.

**Comment by MasterGroosha** [comment]
Please don't close the issue yet, the `env_dist` has incorrect variable naming. Definitely need a fix from my side.

**Comment by MasterGroosha** [comment]
Fixed in d30d4b3c05ba337c9f0d20b2321b6f28be0c3d95

Thank you for reporting!
